next/src/global/Seo.ts

```diff
@@ -1,2 +1,74 @@
-export const domain = "https://www.slowa-klucze.pl";
-export const locale = "pl_PL";
\ No newline at end of file
+import { type global } from '@/types/_pages/global';
+import { sanityFetch } from '@/utils/sanity-client';
+
+export const domain = 'https://www.slowa-klucze.pl';
+export const locale = 'pl_PL';
+
+const SEO = async ({
+  title,
+  description,
+  ogImage,
```

Comment:
Why you declare ogImage for SEO helper util, when in Sanity you doesn't have the ability to pick ogImage for each page individually?